ztp/gitops-subscriptions/argocd/example/optional-extra-manifest/ipsec/README.md

```diff
@@ -53,10 +47,10 @@ interfaces:
     type: tunnel
 ```
 
-Run the build script to create the MachineConfig:
+Run the build script that creates a MachineConfig to import the external certs and apply the IPsec configuration:
 
 `./build.sh`
 
-Apply the MachineConfig:
+### Deploying IPsec encryption
 
-`oc apply -f 99-ipsec-master-endpoint-config.yaml`
+Include `enable-ipsec.yaml` and `99-ipsec-master-endpoint-config.yaml` files in the additional install-time manifests folder set in the `extraManifestPath` parameter in the SiteConfig.
```

Comment:
In 4.14 and later, there is a newer more flexible mechanism in SiteConfig called `extraManifests.searchPaths`.  Let's mention that instead, and then provide a link to the top-level README.md in  ztp/gitops-subscriptions/argocd/README.md that explains more about this mechanism and how to use it.